src/screens/FeedScreen/RenderList.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import PropTypes from 'prop-types';
 import React from 'react';
+import moment from 'moment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `moment` library is a heavy package and can increase the bundle size of your application. If you're only using it for simple date manipulations, consider using lighter alternatives like `date-fns` or native JavaScript Date methods.
```diff
- import moment from 'moment';
+ import { differenceInMilliseconds } from 'date-fns';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->